Frameworks/SPMySQLFramework/TimeZone.h

```diff
@@ -0,0 +1,22 @@
+//
+//  TimeZone.h
+//  SPMySQL.framework
+//
+//  Created by Robin Kunde on 8/19/20.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
`NONNULL` - are we sure `timeZoneIdentifier` can't be nil? :)

Reply:
Good catch.

Reply:
@robinkunde Has this one been resolved?

Reply:
Yes, I've added nullability modifiers.